### DISCUSSION TITLE
Compatibility table

### DISCUSSION THREAD

**Discussion by asarkar** [open-telemetry/opentelemetry-java-instrumentation#2337] [Q&A]
Can we have a compatibility table published for the groups `io.opentelemetry` and `io.opentelemetry.instrumentation`, especially given all the breaking changes happening? According to [this comment](https://github.com/open-telemetry/opentelemetry-java/discussions/2887#discussioncomment-382416), the two groups must currently use the same version, which at the time of this writing, is latest `0.16.0`. There's an obscure reference to this here https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/manual-instrumentation.md#dependencies, but it's for the Java agent, not for the libraries. Not to mention finding such a disclaimer in a page for manual instrumentation is finding a needle in a haystack.

See https://github.com/open-telemetry/opentelemetry-java/discussions/2887 for context.

**Comment by anuraaga** [comment]
Hi @asarkar - we will publish a BOM file with the next release of instrumentation, going forward you should be able to use it and not specify any version numbers for the two groups. After the SDK hits 1.0 end of this month, there should generally not be a problem bumping it ahead of instrumentation but we'd still generally encourage our BOM to prevent problems.

This repo will release 1.0 of the javaagent early next month but the instrumentation libraries are still months away from being stable, and will indicate this with the `-alpha` suffix on the version number. Version alignment should still be fine when using the BOM but warning that there will still be lots of rough times ahead if using our pre-1.0 / alpha artifacts :)
